service/job/job_config.py

```diff
@@ -25,19 +29,21 @@ def _read_job_config_file(file_name):
         return yaml.load(file)
     except Exception as err:
         raise ConfigParseException(
-            "Error while reading job type definition from %s: %s"
-            % (file_name, err))
+            f"Error while reading job type definition from {file_name}: {err}"
```

Comment:
Vielleicht sollten wir uns generell mal einigen ob wir f-Strings nutzen oder %... Du hast das hier zum Beispiel in nen f-String geändert und alles andere aber mit % gelassen.

Reply:
Ja stimmt, prinzipiell finde ich f-Strings auch schöner. Ich habe angefangen es da zu ändern, wo es mir zufällig aufgefallen ist. Wir sollten bei Gelegenheit aber mal durch den Code gehen und es systematisch ändern.

Ticket ist angelegt: http://dai-softsource.uni-koeln.de/issues/8625

Reply:
Gut, dann machen wir das in Zukunft einheitlich